nil/services/synccommittee/prover/tracer/tracer.go

```diff
@@ -29,7 +30,7 @@ type RemoteTracerImpl struct {
 var _ RemoteTracer = new(RemoteTracerImpl)
 
 type TraceConfig struct {
-	ShardID      types.ShardId
+	ShardIDs     []types.ShardId
 	BlockIDs     []transport.BlockReference
```

Comment:
If these two fields should correspond to each other, better to make `[]struct{ShardID types.ShardId, BlockIDtransport.BlockReference}`